verification.go

```diff
@@ -141,7 +141,7 @@ func VerifyBasicBlockFilter(filter *gcs.Filter, block *btcutil.Block) (int,
 			}
 
 			if !match {
-				return 0, fmt.Errorf("filter for block %v is "+
+				log.Errorf("filter for block %v might be "+
```

Comment:
Since this will undoubtedly lead to questions/issues, should we add something along the lines of "might contain taproot outputs which aren't supported yet"?

Reply:
good idea. will add 👍